zerver/openapi/zulip.yaml

```diff
@@ -1859,11 +1859,16 @@ paths:
                                     filters][linkification-filters] that match content in the
                                     message's topic.)
 
-                                    **Changes**: New in Zulip 3.0 (feature level 1).
-                                    Previously, this field was called `subject_links`;
-                                    clients are recommended to rename `subject_links`
-                                    to `topic_links` if present for compatibility with
-                                    older Zulip servers.
+                                    **Changes**: New in Zulip 4.0 (feature level 38).
+                                      Previously, this field returned a list of urls. Now,
+                                      it contains a list of dictionaries containing the matched
+                                      url and text for each link in the `topic`.
```

Comment:
The phase "New in Zulip ..." is meant to be used only when the field is new.  I think we can replace this line with:

"**Changes**: This field contained a list of URLs before Zulip 4.0 (feature level 40)."  We don't need to repeat the details of the new format, because that's explained in the main text of the documentation; we just need to explain what the old format meant.

And then I like your approach of keeping the old Changes as a new paragraph.

Reply:
I think you ended up collapsing the old changes into the existing paragraph; I undid that (see the screenshot in the issue I opened below).